externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/KyuubiSparkUtil.scala

```diff
@@ -90,4 +90,59 @@ object KyuubiSparkUtil extends Logging {
     val runtimeSparkVer = org.apache.spark.SPARK_VERSION
     Utils.majorMinorVersion(runtimeSparkVer)
   }
+
+  /**
+   * Given a Kyuubi/Spark/Hive version string,
+   * return the (major version number, minor version number).
+   * E.g., for 2.0.1-SNAPSHOT, return (2, 0).
+   */
+  def majorMinorVersion(version: String): (Int, Int) = {
```

Comment:
It can be simplified since https://github.com/apache/incubator-kyuubi/pull/3181 get merged.